inherited_resources.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.1'
 
   s.add_dependency("responders")
-  s.add_dependency("actionpack", ">= 3.2", "< 5.1")
-  s.add_dependency("railties", ">= 3.2", "< 5.1")
+  s.add_dependency("actionpack", ">= 3.2", "< 5.2")
```

Comment:
Should this be "< 5.2.x"?  < 5.1 was compatible with 5.1.rc2 but not 5.1.0 which was why we did not update sooner.